src/Glpi/Console/Database/InstallCommand.php

```diff
@@ -303,14 +304,36 @@ public function __construct($dbh)
         }
 
         $output->writeln(
-            '<comment>' . __('Loading default schema...') . '</comment>',
+            '<comment>' . __('Initializing database tables and default data...') . '</comment>',
             OutputInterface::VERBOSITY_VERBOSE
         );
 
+        $progress_bar = new ProgressBar($output);
+        $progress_bar->setFormat('[%bar%] %percent:3s%%' . PHP_EOL . '<comment>%message%</comment>' . PHP_EOL);
+        $progress_bar_started = false;
+
         try {
             $this->db->connect(); // Reconnect DB to ensure it uses update configuration (see `self::configureDatabase()`)
-            Toolbox::createSchema($default_language, $this->db);
+
+            $progress_callback = static function (int $current, ?int $max = null, ?string $data = null) use ($progress_bar, &$progress_bar_started) {
+                if (!$progress_bar_started) {
+                    $progress_bar->setMessage(''); // Empty message on iteration start
+                    $progress_bar->start();
+                    $progress_bar_started = true;
+                }
+                if ($max !== null) {
+                    $progress_bar->setMaxSteps($max);
+                }
+                if ($data !== null) {
+                    $progress_bar->setMessage($data);
+                }
+                $progress_bar->advance($current - $progress_bar->getProgress());
+            };
+
+            Toolbox::createSchema($default_language, $this->db, $progress_callback);
```

Comment:
I think the `$progress_bar_started` is unnecessary

```suggestion

            $progress_callback = static function (int $current, ?int $max = null, ?string $data = null) use ($progress_bar) {
                if ($max !== null) {
                    $progress_bar->setMaxSteps($max);
                }
                if ($data !== null) {
                    $progress_bar->setMessage($data);
                }
                $progress_bar->advance($current - $progress_bar->getProgress());
            };

            $progress_bar->setMessage(''); // Empty message on iteration start
            $progress_bar->start();
            Toolbox::createSchema($default_language, $this->db, $progress_callback);
```